refinery/tool_manager/views.py

```diff
@@ -214,8 +220,11 @@ def _proxy_view(self, request, container_name, url):
             container_name=container_name
         )
         if not user_has_access_to_tool(request.user, visualization_tool):
-            return HttpResponseForbidden(
-                "User does not have permission to view Tool: {}".format(
+            return render_vis_tool_user_not_allowed_template(
+                request,
+                visualization_tool.name,
+                "User: {} does not have permission to view Tool: {}".format(
+                    request.user.username,
                     visualization_tool.uuid
```

Comment:
Would the tool name be more useful than the tool_uuid?

Reply:
We still have the Tool name listed in the sub header. I figured it was useful to include something unique about the Tool that erred to help someone debug if they got a screen shot from an end user

Reply:
That's a fair point. I'd recommend including the tool name in the main body also. We are getting rid of the subheader soon (1-2 milestones).